services/movie_session.py

```diff
@@ -42,3 +42,10 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
The type annotation `list[dict]` is used, which is only supported in Python 3.9 and later. If you need to support earlier versions, consider using `List[Dict]` from the `typing` module.